src/panels/lovelace/special-rows/hui-attribute-row.ts

```diff
@@ -57,7 +60,17 @@ class HuiAttributeRow extends LitElement implements LovelaceRow {
     return html`
       <hui-generic-entity-row .hass=${this.hass} .config=${this._config}>
         <div>
-          ${this._config.prefix} ${attribute || "-"} ${this._config.suffix}
+          ${this._config.prefix}
+          ${this._config.format
```

Comment:
We are now assuming the attribute value is an iso date string when the `format` option is set?

Reply:
That was my thought but open to more generalized ideas

Reply:
That would be in line with what we do for the entity's state output so far according to the docs: https://www.home-assistant.io/lovelace/entities/#format

I had however thought about this topic a bit in the last days (unrelated to this issue/PR here): Ideally the backend integration could provide some type hints (similar to the device class for the entity). Those could then also be used to render the attributes on the more-info dialog in a more appropriate way (rather than e.g. raw timestamps as we do currently). Also, the integration could mark some attributes as "internal" and not have the frontend render them, since I have seen integrations that provide the complete JSON payload that they receive from an external service as an extra attribute, despite breaking out all its values into separate attributes as well.

Was that idea ever discussed with the core repo? Worth creating an architecture issue for?